kolibri/deployment/default/cache.py

```diff
@@ -16,19 +16,21 @@
 diskcache_location = os.path.join(KOLIBRI_HOME, "process_cache")
 
 
-def recreate_cache():
-    shutil.rmtree(diskcache_location, ignore_errors=True)
-    os.mkdir(diskcache_location)
-    diskcache_cache = Cache(diskcache_location, disk_pickle_protocol=pickle_protocol)
-    diskcache_cache.close()
-    return diskcache_cache
-
-
-try:
-    diskcache_cache = Cache(diskcache_location, disk_pickle_protocol=pickle_protocol)
-    diskcache_cache.close()
-except DatabaseError:
-    diskcache_cache = recreate_cache()
+def recreate_diskcache():
+    if cache_options["CACHE_BACKEND"] != "redis":
+        try:
+            diskcache_cache = Cache(
+                diskcache_location, disk_pickle_protocol=pickle_protocol
+            )
+        except DatabaseError:
+            shutil.rmtree(diskcache_location, ignore_errors=True)
+            os.mkdir(diskcache_location)
+            diskcache_cache = Cache(
+                diskcache_location, disk_pickle_protocol=pickle_protocol
+            )
+        diskcache_cache.clear()
+        diskcache_cache.close()
```

Comment:
same as the previous question, not needed if we are not writing on the /.kolibri/process_cache/cache.db database.